lib/complate.js

```diff
@@ -11,9 +11,7 @@ let bundle = new VirtualBundle("./", {
 	browsers: {}
 });
```

Comment:
aside (doesn't really belong here, but ... whatevs?): turns out we only want to obfuscate `Fragment` within `complateRendering.js` ( 🚨 file-name violation), _not_ `createElement`

```javascript
import Renderer, { Fragment as _F, createElement } from "complate-stream";
```

Reply:
I tried this and it doesn't work, because we import createElement in both `components/button/index.jsx` and `components/button/doc.md`. Should I remove the imports there?

Reply:
No, you're right: Seems I was in the wrong frame of mind earlier. Importing complate-stream within snippets might seem redundant, but it's only explicit (AKA correct).